packages/shared/src/components/post/write/WriteLinkPreview.tsx

```diff
@@ -51,16 +71,31 @@ export function WriteLinkPreview({
               </span>
             )}
           </div>
-          <Image className={previewImageClass} src={preview.image} />
-          <Button
-            icon={<OpenLinkIcon />}
-            className="btn-tertiary"
-            type="button"
-            tag="a"
-            target="_blank"
-            rel="noopener noreferrer"
-            href={link}
+          {preview.source && sourceInfoFormat === 'avatar' && (
+            <div className="flex absolute right-28 justify-center items-center w-8 h-8 rounded-full bg-theme-bg-primary">
```

Comment:
Are these styles not already applied on the SourceAvatar?
What was the difference here